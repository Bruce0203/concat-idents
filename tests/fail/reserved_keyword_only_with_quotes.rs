concat_idents::concat_idents!(fn_name = "struct" {
    fn fn_name() {
        /* reserved key-words cannot make up an ident on their own */
    }
});

fn main() {}
