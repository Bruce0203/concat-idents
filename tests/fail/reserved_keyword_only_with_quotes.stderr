error: expected identifier, found keyword `struct`
 --> $DIR/reserved_keyword_only_with_quotes.rs:1:41
  |
1 | concat_idents::concat_idents!(fn_name = "struct" {
  |                                         ^^^^^^^^ expected identifier, found keyword
  |
help: you can escape reserved keywords to use them as identifiers
  |
1 | concat_idents::concat_idents!(fn_name = r#struct {
  |                                         ^^^^^^^^
